hub/api/tensor.py

```diff
@@ -24,11 +24,12 @@ def __init__(
         Args:
             key (str): The internal identifier for this tensor.
             provider (StorageProvider): The storage provider for the parent dataset.
-            tensor_slice (slice): The slice object restricting the view of this tensor.
+            index: The Index object restricting the view of this tensor.
```

Comment:
nit-pick: "The Index object is used to restrict the view of the tensors."

Reply:
Why? This seems to go against the phrasing of existing docstring arguments. What is it that you want to change?